src/container/Maintenance/Maintenance.tsx

```diff
@@ -1,14 +1,18 @@
 import styles from './Maintenance.module.css'
 
+const maintenanceText = process.env.REACT_APP_MAINTENANCE_TEXT || 'xx'
```

Comment:
guess an empty string is a better default if you check `maintenanceText.length > 0` ?

Reply:
For sure! Hacked this together during the call with BTE 🙈 be0eb98078291aeddbc8e95ff1ff8b45c0c4e2f3